src/WSRequestHandler_General.cpp

```diff
@@ -345,3 +345,69 @@ RpcResponse WSRequestHandler::OpenProjector(const RpcRequest& request) {
 	obs_frontend_open_projector(type, monitor, geometry, name);
 	return request.success();
 }
+
+/**
+* Executes hotkey routine, identified by hotkey unique name
+*
+* @param {String} `name` Unique name of the hotkey, as defined when registering the hotkey (e.g. "ReplayBuffer.Save")
+*
+* @api requests
+* @name ProcessHotkeyByName
+* @category general
+* @since unreleased
+*/
+RpcResponse WSRequestHandler::ProcessHotkeyByName(const RpcRequest& request) {
+	const char* name = obs_data_get_string(request.parameters(), "name");
+
+	obs_hotkey_t* hk = Utils::FindHotkeyByName(name);
+	if (!hk) {
+		return request.failed("Hotkey not found");
+	}
+	obs_hotkey_trigger_routed_callback(obs_hotkey_get_id(hk), true);
+	return request.success();
+}
+
+/**
+* Executes hotkey routine, identified by bound combination of keys. A single key combination might trigger multiple hotkey routines depending on user settings 
+*
+* @param {String} `key` Main key identifier (e.g. `OBS_KEY_A` for key "A"). Available identifiers [here](https://github.com/obsproject/obs-studio/blob/master/libobs/obs-hotkeys.h)
+* @param {Object (Optional)} `modifiers` Optional key modifiers object: `{"shift":true/false, "alt":true/false, "control":true/false, "command":true/false}`. False entries can be ommitted
```

Comment:
I would suggest the correct object document format, but it would be easier for me to just commit it. Mind if I do that?

Reply:
Sure, go ahead.
This is my first commit to the project and I tried to stick to your formats as much as I could, but bear with me and make all the changes you consider appropriate.

Reply:
Yeah all good, we don't ever expect people to be able to get it right the first time. obs-studio's PR guidelines are on another level of strict lol